client/package.json

```diff
@@ -25,7 +25,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
```

Comment:
Just a note for @WolfgangDaniel :
Sandro seems to have already ejected our app here (which means that we "unpacked" the used modules of our base (create-react-app). So we should have no more problems with the eject in the design-issues later on!